Makefile

```diff
@@ -41,6 +41,7 @@ test: dependencies
 	go test -v `go list ./... | grep -v caddy-copy`
 
 image-build:
+	if [ ! -f $(BIN) ]; then make docker-build; fi
```

Comment:
We might run the image-build step anyway or else we might end up with an older binary in the container image or?